src/main/java/task/TaskDescription.java

```diff
@@ -0,0 +1,115 @@
+package task;
+
+import java.util.List;
+
+import task.tasks.Task;
+
+/**
+ * Renders description of a task.
+ */
+public class TaskDescription {
+    /**
+     * Renders all outstanding tasks.
+     *
+     * @param tasks Outstanding tasks.
+     */
+    public static String showAllTask(List<Task> tasks) {
+        String result = "";
+        System.out.println("\t____________________________________________________________\n");
+        result += "\t_______________________\n";
+        for (int i = 0; i < tasks.size(); i++) {
+            int serialNumber = i + 1;
+            Task task = tasks.get(i);
+            System.out.println("\t" + serialNumber + "." + task);
+            result += "\t" + serialNumber + "." + task + "\n";
+        }
+        System.out.println("\t____________________________________________________________\n");
+        result += "\t_______________________\n";
+        return result;
+    }
+
+    /**
+     * Renders successful addition of task to outstanding tasks.
+     *
+     * @param tasks   Outstanding tasks.
+     * @param newTask Added task.
```

Comment:
Perhaps it would be better if the space between the parameter name and description is consistent?